cnf-certification-test/identifiers/identifiers.go

```diff
@@ -997,4 +1001,12 @@ that there are no changes to the following directories:
 		BestPracticeReference: bestPracticeDocV1dot3URL + " Section 10.6",
 		ExceptionProcess:      NoDocumentedProcess,
 	},
+	TestNoSSHDaemonsAllowedIdentifier: {
+		Identifier:            TestNoSSHDaemonsAllowedIdentifier,
+		Type:                  normativeResult,
+		Description:           formDescription(TestNoSSHDaemonsAllowedIdentifier, `Check that pods do not SSH daemons.`),
```

Comment:
Check that pods do not SSH daemons. -> Check that pods do not run SSH daemons.

Reply:
Done.